app/views/incomes/HistoricIncomeCalculationView.scala.html

```diff
@@ -15,123 +15,142 @@
  *@
 
 @import includes.ptaHeader
+@import includes.link
+@import uk.gov.hmrc.play.views.html.helpers._
 @import uk.gov.hmrc.tai.config.ApplicationConfig
 @import uk.gov.hmrc.tai.model.domain.{Available, TemporarilyUnavailable}
 @import uk.gov.hmrc.tai.model.TaxYear
 @import uk.gov.hmrc.tai.viewModels.HistoricIncomeCalculationViewModel
 @import uk.gov.hmrc.tai.util.{TaxYearRangeUtil => Dates}
 @import controllers.auth.AuthenticatedRequest
 
-@this(mainTemplate: MainTemplate)
+@this(main: MainTempleteNew)
 
 @(historicIncomeCalculationViewModel: HistoricIncomeCalculationViewModel
 )(implicit request: AuthenticatedRequest[_], messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec : scala.concurrent.ExecutionContext)
 
-@printLink= {
-    <p class="flush--bottom">
-        @includes.link(
-            id=Some("printLink"),
-            url=routes.YourIncomeCalculationController.printYourIncomeCalculationHistoricYears(historicIncomeCalculationViewModel.taxYear,historicIncomeCalculationViewModel.employmentId).toString,
-            copy=messages("tai.label.print"),
-            linkClasses=Seq("tai-print-link"))
-    </p>
-}
-
 @header = @{
     ptaHeader(
+        displayBackLink = false,
         mainHeadingText = messages("tai.income.calculation.TaxableIncomeDetails", historicIncomeCalculationViewModel.employerName.getOrElse("")),
         preHeadingText = messages("tai.yourIncome.preHeading"),
-        displayBackLink = true,
-        additionalTopLinkContent = Some(printLink)
-    )}
+        showGdsVersion = true)
+}
 
-@mainTemplate(
+@main(
     title = messages("tai.yourIncome.heading"),
     authedUser = Some(request.taiUser),
-    pageHeader = Some(header),
-    employerName = historicIncomeCalculationViewModel.employerName
+    backLinkUrl = Some(routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url),
+    backLinkContent = Messages("tai.back-link.upper"),
+    pagePrintable = true,
+    pagePrintName = Some(messages("tai.label.print"))
```

Comment:
pagePrintName purpose is just so we can target this page specifically if we need to in  the css file  - it doesn't serve any visual  on screen purpose  - s o it can be hard coded
example: 
 body.printable-page.XXXXXXX .govuk-footer,

Reply:
should I not set it then?